content/applications/hr/lunch/management.rst

```diff
@@ -0,0 +1,189 @@
+==============
+Managing Lunch
+==============
+
+In Odoo's *Lunch* application, it is required to have someone manage the orders, vendors, and
+products. In addition, someone must be responsible for the orders, and notifying employees when
+their orders have arrived. This can be the same person.
+
+Orders can be :ref:`canceled <lunch/cancel>`, :ref:`sent to the vendor <lunch/send-orders>`,
+:ref:`confirmed <lunch/confirm-orders>` upon arrival, and :ref:`employees can be notified
+<lunch/notify>`, from the :ref:`Today's Orders <lunch/todays-orders>` dashboard, or the
+:ref:`Control Vendors <lunch/control_vendors>` dashboard.
+
+To manage the *Lunch* app, users need :guilabel:`Administrator` rights, set by navigating to the
+:menuselection:`Settings app` and selecting :guilabel:`Manage Users`. Then, click on the desired
+user to view their access rights.
+
+For more information on access rights, refer to the :doc:`Access rights
+</applications/general/users/access_rights/>` documentation.
+
+.. note::
+   Only users with administration rights are able to view the :guilabel:`Manager` and
+   :guilabel:`Configuration` menus.
+
+.. _lunch/todays-orders:
+
+Today's Orders
+==============
+
+To view and manage the orders for the day, navigate to :menuselection:`Lunch app --> Manager -->
+Today's Orders`. All orders for the day are presented in a list view on the :guilabel:`Today's
+Orders` dashboard, with a filter for :guilabel:`Today`, and grouped by :guilabel:`Vendor`, by
+default.
+
+The following information appears in the list:
+
+- :guilabel:`Order Date`: the date the order was placed.
+- :guilabel:`Vendor`: the vendor the product is being ordered from.
+- :guilabel:`Product`: the specific product ordered.
+- :guilabel:`Extras`: any extras selected for the product.
+- :guilabel:`Notes`: any information needed to be sent to the vendor.
+- :guilabel:`User`: the user who ordered the product.
+- :guilabel:`Lunch Location`: where the product is to be delivered.
+- :guilabel:`Price`: the total price for the product, including all extras.
+- :guilabel:`Status`: the current status of the product.
+- :guilabel:`Company`: the company the order was placed under. This only appears in a multi-company
+  database.
+
+.. image:: management/today.png
+   :align: center
+   :alt: The list that appears in the Today's Orders dashboard, with the filters and top column
+         names highlighted.
+
+.. _lunch/cancel:
+
+Cancel orders
+-------------
+
+All users can cancel an order, not just managers of the *Lunch* app.
+
+To cancel an order from a vendor, individual products can be canceled one at a time.
```

Comment:
Is the intent of this sentence to be: "to cancel an order from a vendor, individual products **must** be canceled one at a time"?

If so, you may want to make that adjustment.

If not, you may want to consider a quick rewording of this sentence for easier readability/comprehension.

Reply:
Yes- that's what I mean! You can't cancel a whole order (not sure why) but each individual product.